src/components/registerForm.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+    <div class="mx-auto p-5 rounded-3xl w-min" style="background-color: #313030;">
```

Comment:
Should have same dimensions as design. Overall it should be more inline with design

Reply:
Changes added